pkg/agent/agent.go

```diff
@@ -730,6 +730,15 @@ func (i *Initializer) setupGatewayInterface() error {
 	if err := i.setInterfaceMTU(i.hostGateway, i.networkConfig.InterfaceMTU); err != nil {
 		return err
 	}
+	// Fix #5451
```

Comment:
Typically no need to link issue in code file when the comment can explain the code. Otherwise the code file would be full of issue links.

Reply:
Removed issue link.